R/run_episode_file.R

```diff
@@ -8,8 +8,18 @@
 #' @return a [tibble][tibble::tibble-package] containing the episode file
 #' @export
 #'
-run_episode_file <- function(processed_data_list, year, write_to_disk = TRUE) {
+run_episode_file <- function(processed_data_list, year, write_to_disk = TRUE, ni_only = FALSE) {
+  # Bring all the datasets together from Jen's process functions
   episode_file <- dplyr::bind_rows(processed_data_list) %>%
+    {
+      if (ni_only) {
+        # Drop datasets not needed for NIs ()
+        dplyr::filter(., .data$recid %in% c("01B", "02B", "04B", "GLS", "00B", "PIS")) %>%
+          dplyr::select(dplyr::where(~ any(!is.na(.x))))
+      } else {
+        .
+      }
+    } %>%
```

Comment:
Needs to be re-tested to see how this code interacts with the `store_ep_file_vars` code.